reference/best_practices/reusable_bundles.rst

```diff
@@ -1,4 +1,98 @@
 Reusable Bundles for Akeneo PIM community
 =========================================
 
-Coming soon!!! :)
+You are an integrator and you want to share your project with the awesome Akeneo PIM community?
+Let's read our advices to make a reusable and documented bundle!
+
+
+What do you want when you create a reusable bundle?
+
+* **Compatibility**: to fit Akeneo PIM features
+* **Maintainability**: to make your code flexible and testable (and tested)
+* **Extensibility**: to allow integrators to use your bundle to fit with their specific needs
+* **Scalability**: to know your application limits and document them
+
+
+Compatibility
+-------------
+
+Your bundle won't be reusable for some customers if you forgot some cases. That is no more an issue if you document compatibilities of your bundle.
+
+Akeneo PIM storage stack comes with MySQL and hybrid MySQL/MongoDB (:doc:`/reference/technical_information/choose_database`).
+Use the ProductQueryBuilder which provides an abstraction layer making both storages work with the same API.
+You can refer to this cookbook for more details and use cases: ":doc:`/cookbook/catalog/product/query`".
```

Comment:
I would add link to http://docs.akeneo.com/latest/cookbook/catalog/product/index.html and http://docs.akeneo.com/latest/cookbook/catalog/common/index.html these API work with both storages